cypress/e2e/gui/basic.cy.js

```diff
@@ -0,0 +1,96 @@
+describe('Basic', () => {
+  beforeEach(() => {
+    Cypress.on('window:before:load', window => {
+      window.document.cookie = 'cookieConsent=accepted'
+    })
+    cy.visit('/')
+  })
+
+  it('keeps the filters when coming back from the customer details view', () => {
+    cy.get('[data-testid="size-filter"]').select('Small')
+    cy.get('[data-testid="industry-filter"]').select('Finance')
+    cy.contains('button', 'View').click()
+    cy.contains('h2', 'Customer Details').should('be.visible')
```

Comment:
```suggestion
    cy.contains('h2', 'Customer Details').should('be.visible')

```

Eu deixaria uma linha em branco aqui, para demonstrar que os passos acima são as pré-condições do teste (_Arrange_ no padrão 3A, o qual veremos em um próximo encontro).